apps/backoffice-v2/src/common/components/molecules/DateRangePicker/DateRangePicker.tsx

```diff
@@ -0,0 +1,79 @@
+import React, { useEffect, useState, useRef } from 'react';
+import { CalendarIcon } from '@radix-ui/react-icons';
+import { formatDate } from '@/common/utils/format-date';
+import { DateRange } from 'react-day-picker';
+import { addMonths } from 'date-fns';
+import { ctw } from '@/common/utils/ctw/ctw';
+import { Button } from '../../atoms/Button/Button';
+import { Calendar } from '../../organisms/Calendar/Calendar';
+import { Popover, PopoverContent, PopoverTrigger } from '@ballerine/ui';
+
+type DateRangePickerProps = {
+  onChange: (range: { start: Date | null; end: Date | null }) => void;
+  value: { start: Date | null; end: Date | null };
+};
```

Comment:
Could also utilize `Omit`, `Pick`, and `extends` with interfaces or type intersections to achieve this type.
```suggestion
type TDateRangePickerProps = {
  onChange: ComponentProps<typeof Calender>['onSelect'];
  value: ComponentProps<typeof Calender>['value'];
  className: ComponentProps<'div'>['className'];
};
```

Reply:
Note the library we use here probably exports something like `DayPickerRangeProps`.